recipes/hxnfly/LICENSE

```diff
@@ -0,0 +1,1420 @@
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark">
+  <head>
+    <meta charset="utf-8">
+  <link rel="dns-prefetch" href="https://github.githubassets.com">
+  <link rel="dns-prefetch" href="https://avatars.githubusercontent.com">
+  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com">
+  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">
+
+
```

Comment:
This file does not look good. Please replace it with the raw file from https://raw.githubusercontent.com/nsls-ii-forge/hxnfly-feedstock/master/recipe/LICENSE.